lib/files.js

```diff
@@ -0,0 +1,54 @@
+const { checkIfPathMatches } = require('./parseOwners')
```

Comment:
This is very cool. I had no idea you can do this with require. :-)

Reply:
:+1: